rest-api/dao/hpo_dao.py

```diff
@@ -54,19 +54,20 @@ def _make_query(self, session, query_def):
                               .order_by(HPO.name),
             _ORDER_BY_ENDING)
 
-  def to_client_json(self, model):
-    return HPODao._to_json(model)
+  def to_client_json(self, model, inactive_sites=False):
+    return HPODao._to_json(model, inactive_sites)
 
   @staticmethod
-  def _to_json(model):
+  def _to_json(model, inactive_sites=False):
```

Comment:
You probably don't have to make inactive_sites optional for the to_json methods on HPODao and OrganizationDao; nothing else calls them.